### ISSUE TITLE
[bug] "permission denied" even with shizuku activated

### ISSUE THREAD

**Issue by fallsdevil** [aistra0528/Hail#156]
This is the second time I've tried to use this app on a device with the following characteristics:

 - Samsung
 - Galaxy A01 Core
 - Android 10 (go edition)
 - Using **shizuku** to operate the **Hail** app.

 And it doesn't work, when I allow Hail to use shizuku and then try to freeze an app (downloaded by me and not from the system), it always shows the "Permission denied" message and I don't understand why.  As an attempt I tried using the "device owner - suspended" command, but it gives an error in termux and in the windows terminal (desktop): `Not allowed to set the device owner because there are already some accounts on the device` and unfortunately, I can't use any of the app's functions.

**Comment by mofongox** [comment]
I have the exact same issue. The weird thing is that those apps were able to freeze when I had them in my main profile, but I get a "Permission denied" message when trying to freeze them in my work profile.
Just to clarify, I moved everything that needs Google Framework to function to the work profile, and removed them from my main profile.
Pixel 7 Pro on GrapheneOS.

**Comment by EbanNox** [comment]
Got the same problem... all of shizuku using filemanaging apps fail to write to a subfoldur in the android/data, some of witch are folder with only read access and no write... any attempts to 'chmod -R 777' at these locations also throws me an error that it's not allowed or permission denied... sins all file managers fail I attempt a chmod operation to these directories using termux and 'rish' but all faill

**Comment by mofongox** [comment]
I ended up getting rid of this altogether, and moving apps I need from play store to a phone I keep off in my nightstand's drawer, which I only turn on when I have no other option.
The enshitification that Google has brought to my life is ridiculous. I'm very happy to not have to deal with them anymore.

**Comment by Barkuti** [comment]
I briefly tested this application on a rooted Z2121 still running AOSP v5.7 Android 8.1, working mode root whatever: permission denied.

By the way I completely agree with you, mofongox, concerning the enshittification that G00gl€ has brought to Android already. Modern versions of Android feel more like a jail than an operating system now, being retardedly and/or ridiculously limited in many ways. I feel repugnance for it now.

**Comment by mofongox** [comment]
Yeah. I'm still using that "2 phones" streamline in my life, but the only thing I keep on the Android phone is each of the banking apps (unfortunately all banking apps in this country are useless without an app).

Everything else has been migrated to FOSS alternatives and self-hosted, which I use on a GrapheneOS phone (daily driver).

I'm just waiting for the broader availability of the FLX1s phone by furilabs to order 1 and be done with the BS Google is pulling on Android.

Yeah, I'll still keep a cheap android phone in my drawer for banking, but that's that.

**Comment by mofongox** [comment]
> You mean “… all b̶a̶n̶k̶i̶n̶g̶ ̶a̶p̶p̶s̶ **banks** in this country are useless without an app” I guess, mofongox.

That's exactly right, but I also blame lazy development on the devs side as well.
 
> By the way mofongox you should still be able to operate with banks using their websites, which is the universal way. Unfortunately certain arses like Revolut seem to feel otherwise. 😒

The problem with the banks, and using them only over a browser here is that they require a token for 2FA that is generated ONLY in their app. The app may even fire up, but it's impossible to sign in to it on anything other than stock Android. It is a pain in the ass.

I'll just keep using my current workaround and see what the future brings.
